charts/delphai-deployment/values.yaml

```diff
@@ -43,6 +43,18 @@ scheduling:
   # (For non-interactive only) Set lower priority (e.g. for scrappers)
   lowPriority: false
 
+autoscaling:
+  maxReplicaCount: 16
+
+  rabbitmq:
+    enabled: false
+
+    # The name of the RabbitMQ queue that this service consumes
+    queueName: "service.{{ $.Release.Namespace }}"
+
+    # Message backlog length to trigger autoscaling
+    value: 20
```

Comment:
I've namespaced all rabbit-related here so we'll add other trigger types simpler